### DISCUSSION TITLE
Legacy network folder

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#694] [Ideas]
WinClassic user TechSalt found a way to enable the legacy Network folder.

Here is the reg file:

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\SOFTWARE\Classes\CLSID\{F02C1A0D-BE21-4350-88B0-7367FC96EF3C}]

[HKEY_CURRENT_USER\SOFTWARE\Classes\CLSID\{F02C1A0D-BE21-4350-88B0-7367FC96EF3C}\InProcServer32]

[HKEY_CURRENT_USER\SOFTWARE\Classes\CLSID\{F02C1A0D-BE21-4350-88B0-7367FC96EF3C}\TreatAs]
@="{208D2C60-3AEA-1069-A2D7-08002B30309D}"

```

I think this belongs to File Explorer sphere.

**Comment by valinet** [comment]
What is the "legacy Network folder"?

**Reply by valinet** [reply]
Nvm, it doesn't work in the default configuration of Windows anyway, one has to disable network discovery aka lots of headaches. I found the thread on Winclassic forums. Why is this useful, btw?

**Comment by Anixx** [comment]
I am afraid of making a mistake, but here is the thread: https://winclassic.boards.net/thread/956/legacy-network-folder?page=1
